src/components/Document/DocumentMatches.vue

```diff
@@ -4,28 +4,28 @@
 			<div
 				class="flex justify-between py-1 text-gray-500 uppercase border-b-2 border-gray-400"
 			>
-				<span class="w-2/3 ml-5"> Súbor </span>
+				<span class="w-2/3 ml-3 md:ml-5 text-xs md:text-lg"> Súbor </span>
 
-				<span class="w-1/3 text-center"> Zhoda so súborom </span>
+				<span class="w-1/3 text-center mr-1 md:mr-0 text-xs md:text-lg"> Zhoda so súborom </span>
 			</div>
 			<div
 				v-for="match in matches"
 				:key="match.name"
-				class="flex justify-between py-5 text-black bg-white border-b-2 border-gray-400"
+				class="flex justify-between py-1 md:py-5 text-sm text-black bg-white border-b-2 border-gray-400"
```

Comment:
You forgot to override this on larger devices. Makes all text terribly small.

![image](https://user-images.githubusercontent.com/27869979/109661411-4045d400-7b6a-11eb-871c-9534349662be.png)